examples/nlp/language_modeling/tuning/megatron_gpt_peft_eval.py

```diff
@@ -127,6 +127,12 @@ def main(cfg) -> None:
         peft_model_cfg.data.test_ds = cfg.model.data.test_ds
         peft_model_cfg.activations_checkpoint_granularity = None
         peft_model_cfg.activations_checkpoint_method = None
+        peft_model_cfg.encoder_seq_length = cfg.model.data.test_ds.max_seq_length
+        if 'use_flash_attention' not in peft_model_cfg:
+            peft_model_cfg.use_flash_attention = False
+
+        if 'use_flash_attention' in cfg.model:
+            peft_model_cfg.use_flash_attention = cfg.model.use_flash_attention
```

Comment:
@ekmb Do we need to include evaluation in this PR?

Reply:
no, you can revert